Minichain_v0/consensus.py

```
@@ -0,0 +1,47 @@
+"""
+consensus.py - Proof-of-Work mining for MiniChain.
+Finds a nonce that makes the block hash start with required zeros.
+"""
+
+import time
+from typing import List
```

Ruff (0.15.1)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)